README.md

```diff
@@ -2151,10 +2151,10 @@ no parameters.
   ```
 
 
-* <a name="camelcase-classes"></a>
-  Use `CamelCase` for classes and modules.  (Keep acronyms like HTTP, RFC, XML
+* <a name="pascalcase-classes"></a>
+  Use `PascalCase` for classes and modules.  (Keep acronyms like HTTP, RFC, XML
```

Comment:
I like that. The distinction as "UpperCamelCase" (as opposed to PascalCase and camelCase) reminds the reader that this isn't your professor's camelCase being discussed here. :)